bin/package_build.py

```diff
@@ -180,6 +185,11 @@ def pds(q):
 		print(f"Extracting {file} from PDS data ... ")
 		pds(file)
 	elif file == 'Debian' or file == 'debian':
+	######
+	#NOTE
+	#For Debian list, we are pulling pkg data regardless whether it's s390x compatible or not.
+	#If you have any resources that can help us narrow down our list, feel free to raise an issue :)
+	#####
```

Comment:
We should remove this comment now, since we're only pulling s390x data now! :tada:

Reply:
Yes yes, will update rhe submodule too!